tests/testthat/helper.R

```diff
@@ -0,0 +1,43 @@
+# Define the URLs
+urls <- c(
+  "https://lw-enram.s3-eu-west-1.amazonaws.com/be/jab/2016/09/19/23/bejab_vp_201609192315.h5",
+  "https://lw-enram.s3-eu-west-1.amazonaws.com/be/jab/2016/09/19/23/bejab_vp_201609192320.h5",
+  "https://lw-enram.s3-eu-west-1.amazonaws.com/cz/brd/2016/09/19/00/czbrd_vp_20160919T0000Z_0x5.h5",
+  "https://aloft.s3-eu-west-1.amazonaws.com/baltrad/monthly/bejab/2023/bejab_vpts_202303.csv.gz",
+  "https://aloft.s3-eu-west-1.amazonaws.com/baltrad/monthly/bejab/2023/bejab_vpts_202304.csv.gz",
+  "https://aloft.s3-eu-west-1.amazonaws.com/baltrad/monthly/bewid/2023/bewid_vpts_202303.csv.gz"
+)
+
+
+# Define the path to the new temporary directory
+temp_dir <- 'temp'
+
+# Create the new directory if not exists
+if (!dir.exists(temp_dir)) {
+  dir.create(temp_dir)
+}
+
+# Create the h5 and csv sub-directories
+h5_dir <- file.path(temp_dir, "h5")
+csv_dir <- file.path(temp_dir, "csv")
+if (!dir.exists(h5_dir)) dir.create(h5_dir)
+if (!dir.exists(csv_dir)) dir.create(csv_dir)
+
+# Function to download a file
+download_test_file <- function(url, dest_dir = temp_dir) {
```

Comment:
This helper.R file should only contain the `download_test_file()` function. Any code above and below is in preparation of the `read_vpts()` tests and should be moved to `test-read_vpts.R`, where the helper function can be called.